main.py

```diff
@@ -1,15 +1,22 @@
 import requests
 import json
 import os
+import pathlib
+from config import *
+import datetime
 
-URL = "https://api.divar.ir/v8/web-search/{SEARCH_CONDITIONS}".format(**os.environ)
+URL = f"https://api.divar.ir/v8/web-search/{SEARCH_CONDITIONS}"
 TOKENS = list()
-BOT_TOKEN = '{BOT_TOKEN}'.format(**os.environ)
-BOT_CHATID = '{BOT_CHATID}'.format(**os.environ)
+BOT_TOKEN = f"{BOT_TOKEN}"
+BOT_CHATID = f"{BOT_CHATID}"
 
 
-def get_data():
-    response = requests.get(URL)
+def get_data(page=None):
+    if page:
+        api_url = URL + f"&page={page}"
+    else:
+        api_url = URL
```

Comment:
I guess it's cleaner this way:
```suggestion
    api_url = URL
    if page:
        api_url += f"&page={page}"
```

Reply:
Agreed